packages/transaction-controller/src/types.ts

```diff
@@ -54,6 +60,24 @@ type TransactionMetaBase = {
    */
   deviceConfirmedOn?: WalletDevice;
 
+  /**
+   * [swaps-only]
```

Comment:
Shall we move this to a second line in the form of `Only used by swaps transactions.` ?